woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -271,10 +284,6 @@ def test_accessor_equality_with_schema(sample_df, sample_column_names, sample_in
     # eq not implemented on Accessor class, so Schema's eq is called
     assert schema_df.ww.__eq__(comparison_schema)
```

Comment:
Maybe something we should think about for the future, but is this adequate now given the changes we have made? If two accessors have different settings for `use_standard_tags` are they equal? If two have the same logical types and semantic tags but different data, are they equal?

Reply:
Good question. I'm thinking that you're right and we should be checking, at the very least, the underlying data as well (probably only for pandas dataframes, though)

For the `use_standard_tags` question, I think two dataframes with opposite `use_standard_tags` values would be equal if no column's logical type had any standard tags.

I'll make an issue, and given that the current equality behavior in the Table Accessor is worse than that in DataTables, we should add it to the initial Schema/Accessor epic

Reply:
Created #686